pyro/contrib/gp/models/sgpr.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.autograd import Variable
+import torch.nn as nn
+
+import pyro
+import pyro.distributions as dist
+from pyro.distributions.util import matrix_triangular_solve_compat
+
+
+class SparseGPRegression(nn.Module):
+    """
+    Sparse Gaussian Process Regression module.
+
+    :param torch.autograd.Variable X: A tensor of inputs.
+    :param torch.autograd.Variable y: A tensor of outputs for training.
+    :param pyro.contrib.gp.kernels.Kernel kernel: A Pyro kernel object.
+    :param pyro.contrib.gp.InducingPoints Xu: An inducing-point module for spare approximation.
+    :param torch.Tensor noise: An optional noise tensor.
+    :param str approx: One of approximation methods: 'DTC', 'FITC', and 'VFE' (default).
```

Comment:
may be nice for newcomers to expand these acronyms

Reply:
I address this in the description of the model. :)